notebooks/image_models/solutions/serving.ipynb

```diff
@@ -0,0 +1,1180 @@
+{
```

Comment:
<p>"... making it useful for sharing or deploying with TFLite, TensorFlow.js, TensorFlow Serving, or TensorFlow Hub" &gt;&gt; "... making it useful for sharing. SavedModel is also highly portable and can be deployed to across many targets like TFLite, TensorFlow.js, Tensorflow Serving, or Tensorflow Hub".</p><p> </p><p> </p><p>"Here let's try to export SavedModel with a high-level API" &gt;&gt; "Here let's export our trained model to a SavedModel with a high-level API." </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Done. Thanks!</p>